src/services/logger/serializers.ts

```diff
@@ -0,0 +1,76 @@
+import * as Hapi from 'hapi';
+import * as Cookie from 'cookie';
+import * as JWT from 'jsonwebtoken';
+import { compose, evolve, omit } from 'ramda';
+import { getConfig } from '../../../config';
+import { Session } from '../../auth/strategies/standard/types';
+
+
+const { auth: { standard } } = getConfig(process.env.NODE_ENV);
+
+const extractCookie = (c: string) => Cookie.parse(c)[standard.cookie.name];
+
+const extractBearerToken = (c: string = '') => c.split('Bearer ')[1] || c || '';
+
+const extractToken = (headers: Hapi.Util.Dictionary<string>) =>
+  headers.cookie
+    ? extractCookie(headers.cookie)
+    : extractBearerToken(headers.authorization);
+
+const decodeToken = (c: string): Partial<Session> => {
+  let decoded: Partial<Session>;
+  try {
+    decoded = <any> JWT.verify(c, standard.jwt.secret, standard.jwt.verifyOptions);
+
+  } catch (error) {
+    decoded = {};
+  }
+
+  return decoded;
+};
+
+const getIds = compose(decodeToken, extractToken);
+
+const attachUserId = (req: { headers: Hapi.Util.Dictionary<string> }) => {
+  const { userId, organisationId } = getIds(req.headers);
+  return { ...req, userId, organisationId };
```

Comment:
`userId` & `organisationId` are used in a number of payloads and responses, so it will be difficult to filter on this. I think we should make a unique string we can search on eg `auth_user:1`